tests/fixture/database/test_postgres.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer
+from sqlalchemy.ext.declarative import declarative_base
+
+from pytest_mock_resources import create_postgres_fixture
+
+Base = declarative_base()
+
+
+class Thing(Base):
+    __tablename__ = "thing"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+
+
+createdb_template_pg = create_postgres_fixture(Base, createdb_template="template0", session=True)
+
+
+def test_createdb_template(createdb_template_pg):
+    """Assert successful usage of a fixture which sets the `createdb_template` argument."""
+    thing = Thing(id=1)
+    createdb_template_pg.add(thing)
+    createdb_template_pg.commit()
```

Comment:
does this test actually test the thing or should we be asserting something here?

Reply:
it tests the thing in the sense that I was making sure our database setup stuff worked and the table exists. I suppose this would also pass if we hadn't actually set `template=template0`, but i dont know of a way to test that safely